common/main/ff_Adlm_SL.xml

```diff
@@ -50,6 +50,9 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 				<symbol>GNF</symbol>
 			</currency>
 			<currency type="SLL">
+				<symbol>Le (old)</symbol>
```

Comment:
Don't we usually do a date range like "Le (1964–2022)" instead of "(old)"?

Reply:
yes, @macchiati had the same comment, will do